### ISSUE TITLE
2025-03-14, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#110]
# Today's Arxiv Papers

(1) [GoT: Unleashing Reasoning Capability of Multimodal Large Language Model for Visual Generation and Editing](http://arxiv.org/abs/2503.10639v1), 2025-03-13

(2) [HybridVLA: Collaborative Diffusion and Autoregression in a Unified Vision-Language-Action Model](http://arxiv.org/abs/2503.10631v1), 2025-03-13

(3) [UniGoal: Towards Universal Zero-shot Goal-oriented Navigation](http://arxiv.org/abs/2503.10630v1), 2025-03-13

(4) [Siege: Autonomous Multi-Turn Jailbreaking of Large Language Models with Tree Search](http://arxiv.org/abs/2503.10619v1), 2025-03-13

(5) [From TOWER to SPIRE: Adding the Speech Modality to a Text-Only LLM](http://arxiv.org/abs/2503.10620v1), 2025-03-13

(6) [Compositional Subspace Representation Fine-tuning for Adaptive Large Language Models](http://arxiv.org/abs/2503.10617v1), 2025-03-13

(7) [R1-Onevision: Advancing Generalized Multimodal Reasoning through Cross-Modal Formalization](http://arxiv.org/abs/2503.10615v1), 2025-03-13

(8) [CoSTA$\ast$: Cost-Sensitive Toolpath Agent for Multi-turn Image Editing](http://arxiv.org/abs/2503.10613v1), 2025-03-13

(9) [TruthPrInt: Mitigating LVLM Object Hallucination Via Latent Truthful-Guided Pre-Intervention](http://arxiv.org/abs/2503.10602v1), 2025-03-13

(10) [Unlock the Power of Unlabeled Data in Language Driving Model](http://arxiv.org/abs/2503.10586v1), 2025-03-13

(11) [Unveiling the Mathematical Reasoning in DeepSeek Models: A Comparative Study of Large Language Models](http://arxiv.org/abs/2503.10573v1), 2025-03-13

(12) [ASIDE: Architectural Separation of Instructions and Data in Language Models](http://arxiv.org/abs/2503.10566v1), 2025-03-13

(13) [Short-term AI literacy intervention does not reduce over-reliance on incorrect ChatGPT recommendations](http://arxiv.org/abs/2503.10556v1), 2025-03-13

(14) [KUDA: Keypoints to Unify Dynamics Learning and Visual Prompting for Open-Vocabulary Robotic Manipulation](http://arxiv.org/abs/2503.10546v1), 2025-03-13

(15) [PiSA: A Self-Augmented Data Engine and Training Strategy for 3D Understanding with Large Models](http://arxiv.org/abs/2503.10529v1), 2025-03-13

(16) [Probing LLMs for Multilingual Discourse Generalization Through a Unified Label Set](http://arxiv.org/abs/2503.10515v1), 2025-03-13

(17) [Conformal Prediction Sets for Deep Generative Models via Reduction to Conformal Regression](http://arxiv.org/abs/2503.10512v1), 2025-03-13

(18) [SySLLM: Generating Synthesized Policy Summaries for Reinforcement Learning Agents Using Large Language Models](http://arxiv.org/abs/2503.10509v1), 2025-03-13

(19) [TokenCarve: Information-Preserving Visual Token Compression in Multimodal Large Language Models](http://arxiv.org/abs/2503.10501v1), 2025-03-13

(20) [MMLU-ProX: A Multilingual Benchmark for Advanced Large Language Model Evaluation](http://arxiv.org/abs/2503.10497v1), 2025-03-13

(21) [Source-primed Multi-turn Conversation Helps Large Language Models Translate Documents](http://arxiv.org/abs/2503.10494v1), 2025-03-13

(22) [LLMs in Disease Diagnosis: A Comparative Study of DeepSeek-R1 and O3 Mini Across Chronic Health Conditions](http://arxiv.org/abs/2503.10486v1), 2025-03-13

(23) [Statistical Analysis of Sentence Structures through ASCII, Lexical Alignment and PCA](http://arxiv.org/abs/2503.10470v1), 2025-03-13

(24) [DynaCode: A Dynamic Complexity-Aware Code Benchmark for Evaluating Large Language Models in Code Generation](http://arxiv.org/abs/2503.10452v1), 2025-03-13

(25) [4D LangSplat: 4D Language Gaussian Splatting via Multimodal Large Language Models](http://arxiv.org/abs/2503.10437v1), 2025-03-13

(26) [BeamLLM: Vision-Empowered mmWave Beam Prediction with Large Language Models](http://arxiv.org/abs/2503.10432v1), 2025-03-13

(27) [Understanding the Logical Capabilities of Large Language Models via Out-of-Context Representation Learning](http://arxiv.org/abs/2503.10408v1), 2025-03-13

(28) [RealGeneral: Unifying Visual Generation via Temporal In-Context Learning with Video Models](http://arxiv.org/abs/2503.10406v1), 2025-03-13

(29) [CINEMA: Coherent Multi-Subject Video Generation via MLLM-Based Guidance](http://arxiv.org/abs/2503.10391v1), 2025-03-13

(30) [SPPO:Efficient Long-sequence LLM Training via Adaptive Sequence Pipeline Parallel Offloading](http://arxiv.org/abs/2503.10377v1), 2025-03-13

(31) [G-Boost: Boosting Private SLMs with General LLMs](http://arxiv.org/abs/2503.10367v1), 2025-03-13

(32) [Do I look like a `cat.n.01` to you? A Taxonomy Image Generation Benchmark](http://arxiv.org/abs/2503.10357v1), 2025-03-13

(33) [New Trends for Modern Machine Translation with Large Reasoning Models](http://arxiv.org/abs/2503.10351v1), 2025-03-13

(34) [KV-Distill: Nearly Lossless Learnable Context Compression for LLMs](http://arxiv.org/abs/2503.10337v1), 2025-03-13

(35) [OSMa-Bench: Evaluating Open Semantic Mapping Under Varying Lighting Conditions](http://arxiv.org/abs/2503.10331v1), 2025-03-13

(36) [Collaborative Speculative Inference for Efficient LLM Inference Serving](http://arxiv.org/abs/2503.10325v1), 2025-03-13

(37) [IDEA: Inverted Text with Cooperative Deformable Aggregation for Multi-modal Object Re-Identification](http://arxiv.org/abs/2503.10324v1), 2025-03-13

(38) [Capturing Semantic Flow of ML-based Systems](http://arxiv.org/abs/2503.10310v1), 2025-03-13

(39) [Test Amplification for REST APIs Using "Out-of-the-box" Large Language Models](http://arxiv.org/abs/2503.10306v1), 2025-03-13

(40) [VisualPRM: An Effective Process Reward Model for Multimodal Reasoning](http://arxiv.org/abs/2503.10291v1), 2025-03-13

(41) [An Expanded Massive Multilingual Dataset for High-Performance Language Technologies](http://arxiv.org/abs/2503.10267v1), 2025-03-13

(42) [Numerical Error Analysis of Large Language Models](http://arxiv.org/abs/2503.10251v1), 2025-03-13

(43) [LLM Agents Display Human Biases but Exhibit Distinct Learning Patterns](http://arxiv.org/abs/2503.10248v1), 2025-03-13

(44) [MinorBench: A hand-built benchmark for content-based risks for children](http://arxiv.org/abs/2503.10242v1), 2025-03-13

(45) [SCOOP: A Framework for Proactive Collaboration and Social Continual Learning through Natural Language Interaction andCausal Reasoning](http://arxiv.org/abs/2503.10241v1), 2025-03-13

(46) [Unveiling the Invisible: Reasoning Complex Occlusions Amodally with AURA](http://arxiv.org/abs/2503.10225v1), 2025-03-13

(47) [Efficient Federated Fine-Tuning of Large Language Models with Layer Dropout](http://arxiv.org/abs/2503.10217v1), 2025-03-13

(48) [Adaptive Inner Speech-Text Alignment for LLM-based Speech Translation](http://arxiv.org/abs/2503.10211v1), 2025-03-13

(49) [LVAgent: Long Video Understanding by Multi-Round Dynamical Collaboration of MLLM Agents](http://arxiv.org/abs/2503.10200v1), 2025-03-13

(50) [PRISM: Preference Refinement via Implicit Scene Modeling for 3D Vision-Language Preference-Based Reinforcement Learning](http://arxiv.org/abs/2503.10177v1), 2025-03-13

(51) ["Well, Keep Thinking": Enhancing LLM Reasoning with Adaptive Injection Decoding](http://arxiv.org/abs/2503.10167v1), 2025-03-13

(52) [Retrieval-Augmented Generation with Hierarchical Knowledge](http://arxiv.org/abs/2503.10150v1), 2025-03-13

(53) [Gumiho: A Hybrid Architecture to Prioritize Early Tokens in Speculative Decoding](http://arxiv.org/abs/2503.10135v1), 2025-03-13

(54) [Hybrid Agents for Image Restoration](http://arxiv.org/abs/2503.10120v1), 2025-03-13

(55) [StepMathAgent: A Step-Wise Agent for Evaluating Mathematical Processes through Tree-of-Error](http://arxiv.org/abs/2503.10105v1), 2025-03-13

(56) [AgentDAO: Synthesis of Proposal Transactions Via Abstract DAO Semantics](http://arxiv.org/abs/2503.10099v1), 2025-03-13

(57) [Cognitive-Mental-LLM: Leveraging Reasoning in Large Language Models for Mental Health Prediction via Online Text](http://arxiv.org/abs/2503.10095v1), 2025-03-13

(58) [Representation-based Reward Modeling for Efficient Safety Alignment of Large Language Model](http://arxiv.org/abs/2503.10093v1), 2025-03-13

(59) [Why Does Your CoT Prompt (Not) Work? Theoretical Analysis of Prompt Space Complexity, its Interaction with Answer Space During CoT Reasoning with LLMs: A Recurrent Perspective](http://arxiv.org/abs/2503.10084v1), 2025-03-13

(60) [Information Density Principle for MLLM Benchmarks](http://arxiv.org/abs/2503.10079v1), 2025-03-13

(61) [VMBench: A Benchmark for Perception-Aligned Video Motion Generation](http://arxiv.org/abs/2503.10076v1), 2025-03-13

(62) [Advanced Tool Learning and Selection System (ATLASS): A Closed-Loop Framework Using LLM](http://arxiv.org/abs/2503.10071v1), 2025-03-13

(63) [SmartWay: Enhanced Waypoint Prediction and Backtracking for Zero-Shot Vision-and-Language Navigation](http://arxiv.org/abs/2503.10069v1), 2025-03-13

(64) [Compute Optimal Scaling of Skills: Knowledge vs Reasoning](http://arxiv.org/abs/2503.10061v1), 2025-03-13

(65) [Enhancing Multi-Agent Systems via Reinforcement Learning with LLM-based Planner and Graph-based Policy](http://arxiv.org/abs/2503.10049v1), 2025-03-13

(66) [How Do Multimodal Large Language Models Handle Complex Multimodal Reasoning? Placing Them in An Extensible Escape Game](http://arxiv.org/abs/2503.10042v1), 2025-03-13

(67) [NumScout: Unveiling Numerical Defects in Smart Contracts using LLM-Pruning Symbolic Execution](http://arxiv.org/abs/2503.10041v1), 2025-03-13

(68) [OR-LLM-Agent: Automating Modeling and Solving of Operations Research Optimization Problem with Reasoning Large Language Model](http://arxiv.org/abs/2503.10009v1), 2025-03-13

(69) [TIME: Temporal-sensitive Multi-dimensional Instruction Tuning and Benchmarking for Video-LLMs](http://arxiv.org/abs/2503.09994v1), 2025-03-13

(70) [From Equations to Insights: Unraveling Symbolic Structures in PDEs with LLMs](http://arxiv.org/abs/2503.09986v1), 2025-03-13

(71) [Faster Inference of LLMs using FP8 on the Intel Gaudi](http://arxiv.org/abs/2503.09975v1), 2025-03-13

(72) [Modeling Thousands of Human Annotators for Generalizable Text-to-Image Person Re-identification](http://arxiv.org/abs/2503.09962v1), 2025-03-13

(73) [Exploring Mutual Empowerment Between Wireless Networks and RL-based LLMs: A Survey](http://arxiv.org/abs/2503.09956v1), 2025-03-13

(74) [UVE: Are MLLMs Unified Evaluators for AI-Generated Videos?](http://arxiv.org/abs/2503.09949v1), 2025-03-13

(75) [PluralLLM: Pluralistic Alignment in LLMs via Federated Learning](http://arxiv.org/abs/2503.09925v1), 2025-03-13

(76) [Conversational Gold: Evaluating Personalized Conversational Search System using Gold Nuggets](http://arxiv.org/abs/2503.09902v1), 2025-03-12

(77) [Improving the Reusability of Conversational Search Test Collections](http://arxiv.org/abs/2503.09899v1), 2025-03-12

(78) [What's In Your Field? Mapping Scientific Research with Knowledge Graphs and Large Language Models](http://arxiv.org/abs/2503.09894v1), 2025-03-12

(79) [LuciBot: Automated Robot Policy Learning from Generated Videos](http://arxiv.org/abs/2503.09871v1), 2025-03-12

(80) [Media and responsible AI governance: a game-theoretic and LLM analysis](http://arxiv.org/abs/2503.09858v1), 2025-03-12

(81) [BioSpark: Beyond Analogical Inspiration to LLM-augmented Transfer](http://dx.doi.org/10.1145/3706598.3714053), 2025-03-12

(82) [SE(3)-Equivariant Robot Learning and Control: A Tutorial Survey](http://arxiv.org/abs/2503.09829v1), 2025-03-12

(83) [Generative AI for Named Entity Recognition in Low-Resource Language Nepali](http://arxiv.org/abs/2503.09822v1), 2025-03-12

(84) [Attention Reveals More Than Tokens: Training-Free Long-Context Reasoning with Attention-guided Retrieval](http://arxiv.org/abs/2503.09819v1), 2025-03-12

(85) [Communication-Efficient Language Model Training Scales Reliably and Robustly: Scaling Laws for DiLoCo](http://arxiv.org/abs/2503.09799v1), 2025-03-12

(86) [Constrained Language Generation with Discrete Diffusion Models](http://arxiv.org/abs/2503.09790v1), 2025-03-12

(87) [AgentDAM: Privacy Leakage Evaluation for Autonomous Web Agents](http://arxiv.org/abs/2503.09780v1), 2025-03-12

(88) [Multi-Agent LLM Actor-Critic Framework for Social Robot Navigation](http://arxiv.org/abs/2503.09758v1), 2025-03-12

(89) [Advancing Education through Tutoring Systems: A Systematic Literature Review](http://arxiv.org/abs/2503.09748v1), 2025-03-12

(90) [Review GIDE -- Restaurant Review Gastrointestinal Illness Detection and Extraction with Large Language Models](http://arxiv.org/abs/2503.09743v1), 2025-03-12

(91) [Local Look-Ahead Guidance via Verifier-in-the-Loop for Automated Theorem Proving](http://arxiv.org/abs/2503.09730v1), 2025-03-12

(92) [Leveraging Social Media and Google Trends to Identify Waves of Avian Influenza Outbreaks in USA and Canada](http://arxiv.org/abs/2503.09725v1), 2025-03-12

(93) [Have LLMs Made Active Learning Obsolete? Surveying the NLP Community](http://arxiv.org/abs/2503.09701v1), 2025-03-12

(94) [MoC: Mixtures of Text Chunking Learners for Retrieval-Augmented Generation System](http://arxiv.org/abs/2503.09600v1), 2025-03-12

(95) [How to Protect Yourself from 5G Radiation? Investigating LLM Responses to Implicit Misinformation](http://arxiv.org/abs/2503.09598v1), 2025-03-12

(96) [Proceedings of the ISCA/ITG Workshop on Diversity in Large Speech and Language Models](http://arxiv.org/abs/2503.10298v1), 2025-03-12

(97) [SimLingo: Vision-Only Closed-Loop Autonomous Driving with Language-Action Alignment](http://arxiv.org/abs/2503.09594v1), 2025-03-12

(98) [BIMBA: Selective-Scan Compression for Long-Range Video Question Answering](http://arxiv.org/abs/2503.09590v2), 2025-03-12

(99) [Cost-Optimal Grouped-Query Attention for Long-Context LLMs](http://arxiv.org/abs/2503.09579v1), 2025-03-12

(100) [Probabilistic Reasoning with LLMs for k-anonymity Estimation](http://arxiv.org/abs/2503.09674v1), 2025-03-12
